django_enum_choices/forms.py

```diff
@@ -32,7 +32,7 @@ def to_python(self, value):
             return
 
         if value in self.empty_values:
-            return ''
+            value = ''
```

Comment:
Apologies for the late response.

It seems like this issue can be solved by removing the whole check since `empty_values = [None, '', [], (), {}]` by default.
I did a quick test in a playground project and it seems to be working properly, I'm not even sure why we placed that check here in the first place.

Reply:
You're right. PR updated.